src/components/browser-notification/BrowserNotification.js

```diff
@@ -81,21 +81,53 @@ const getNewMessageOptions = props => {
   }
 }
 
+const getIncomingCallOptions = props => {
+  const content =
+    props.notification.channel.type === 'room'
+      ? props.intl.formatMessage(
```

Comment:
@nikgraf probably can we move those translations to `messages`? Or in reverse, move translations from `messages` to code. Just want to keep translations under the same conditions. And actually, I think I already used to find translations at start of the file :D

Reply:
initially missunderstood your comment, but just pushed a version that would be ideal for oyu